{"name": "Получение заказа без трек-номера или с несуществующим трек-номером", "status": "passed", "description": "Проверка возможности получения заказа без трек-номера или с несуществующим трек-номером", "parameters": [{"name": "track_number", "value": "9578262"}, {"name": "answer", "value": "404"}], "start": 1701451449779, "stop": 1701451450084, "uuid": "de5af9b8-77f1-4196-b31b-3eb8308985e7", "historyId": "97c5b65967dc534ec3612819affb9b0b", "testCaseId": "a9be370ae517119227050ed7e4fc8b27", "fullName": "tests.test_orders.TestGetOrders#test_get_order_by_track_number_fail", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_orders"}, {"name": "subSuite", "value": "TestGetOrders"}, {"name": "host", "value": "Ya"}, {"name": "thread", "value": "34240-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_orders"}]}